manager/dispatcher/dispatcher_test.go

```diff
@@ -15,16 +15,23 @@ import (
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/credentials"
 
+	"encoding/json"
```

Comment:
nit: Standard library imports such as this are typically put in the top section of the `import` statement, sorted alphabetically.